.github/ISSUE_TEMPLATE/F-submit-statistical-software-for-review.md

```diff
@@ -10,6 +10,7 @@ Other Package Authors: (delete if none) Name (@github_handle)
 Repository:  <!--repourl--><!--end-repourl-->
 Version submitted:
 Submission type: <!--submission-type-->Stats<!--end-submission-type-->
+Badge grade: <!--statsgrade-->bronze/silver/gold (select one)<!--end-statsgrade-->
```

Comment:
Just to be sure, this means submitters choose one grade goal at submission?

Reply:
Yep, that's right. There is currently no check for this, because we really need to chat with @xuanxu about where might be the best place to implement checks on expected values of variables. That shouldn't stop us merging, because things can always be manually edited in the meantime anyway.